src/components/EditSections/EditUserInfo/components/ProfilePicture/ProfilePicture.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormattedMessage, useIntl } from 'react-intl';
+
+import {
+  Button,
+  Dropdown,
+  DropdownMenu,
+  Icon,
+  Label,
+} from '@folio/stripes/components';
+import { useStripes } from '@folio/stripes/core';
+
+import { useProfilePicture } from '../../../../../hooks';
+import { isAValidUUID } from '../../../../util/util';
+import profilePicThumbnail from '../../../../../../icons/profilePicThumbnail.png';
+import css from '../../EditUserInfo.css';
+
+const ProfilePicture = ({ label, profilePictureLink }) => {
+  const intl = useIntl();
+  const stripes = useStripes();
+  const isProfilePictureLinkAURL = !isAValidUUID(profilePictureLink);
+  const hasProfilePicture = Boolean(profilePictureLink);
+  const { isFetching, profilePictureData } = useProfilePicture({ profilePictureId: profilePictureLink });
+  const hasAllProfilePicturePerms = stripes.hasPerm('ui-users.profile-pictures.all');
+
+  const renderProfilePic = () => {
+    const profilePictureSrc = isProfilePictureLinkAURL ? profilePictureLink : 'data:;base64,' + profilePictureData;
+    const imgSrc = isFetching || !hasProfilePicture ? profilePicThumbnail : profilePictureSrc;
+
+    return (
+      <img
+        data-testid="profile-picture"
+        className={css.profilePlaceholder}
+        alt={intl.formatMessage({ id: 'ui-users.information.profilePicture' })}
+        src={imgSrc}
+      />
+    );
+  };
+
+  const renderMenu = () => (
+    <DropdownMenu
+      aria-label="profile picture action menu"
+      role="menu"
+    >
+      <Button buttonStyle="dropdownItem"><Icon icon="profile">Local File</Icon></Button>
+      <Button buttonStyle="dropdownItem"><Icon icon="external-link">External URL</Icon></Button>
+      <Button buttonStyle="dropdownItem"><Icon icon="trash">Delete</Icon></Button>
+    </DropdownMenu>
+  );
+
+  return (
+    <>
+      <Label tagName="div">
+        {label}
+      </Label>
+      { renderProfilePic()}
```

Comment:
Should this be `<Label>{renderProfilePic()}</Label>` instead? It's a bit odd to see a `<Label>` all on its own, unattached to a field since its raison d'être is to label a field.